lib/portlayer/portlayer.go

```diff
@@ -57,5 +58,13 @@ func Init(ctx context.Context, sess *session.Session) error {
 	extraconfig.Decode(source, &storage.Config)
 	log.Debugf("Decoded VCH config for storage: %#v", storage.Config)
 
+	// create or restore a portlayer k/v store
+	if len(storage.Config.ImageStores) > 0 {
+		// init the store package and create the default store
+		if err = store.Init(ctx, sess, storage.Config.ImageStores[0]); err != nil {
```

Comment:
Q: why are we initializing based on image store presence given it's scope is VCH, not image store?